apps/web/Dockerfile

```
@@ -1,21 +1,55 @@
-FROM oven/bun:alpine
-
+# Builder stage
+FROM oven/bun AS builder
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)